platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/conflicts/VersionConflictException.java

```diff
@@ -64,37 +57,8 @@ private static String buildMessage(Collection<Conflict> conflicts) {
         return formatter.toString();
     }
 
-    private static String getDependencyNotation(Collection<Conflict> conflicts) {
-        return conflicts.stream()
-            .findFirst()
-            .map(p -> {
-                ModuleVersionIdentifier identifier = p.getVersions().get(0);
-                return identifier.getGroup() + ":" + identifier.getName();
-            })
-            .orElseThrow(() -> new GradleException("This "));
-    }
-
-    private static List<String> createResolutions(String projectPath, String configurationName, String dependencyNotation) {
-        if (projectPath.equals(":")) {
-            projectPath = "";
-        }
-
-        return ImmutableList.of("Run with " + projectPath + ":dependencyInsight --configuration " +
-            configurationName + " --dependency " + dependencyNotation + " to get more insight on how to solve the conflict.");
-    }
-
     @Override
     public List<String> getResolutions() {
         return resolutions;
     }
-
-    public static VersionConflictException create(
```

Comment:
Logic moved to `ConfigurationFailureContext#getResolutionsForVersionConflict` in `DefaultConfiguration`